server/trillian_log_server/main.go

```diff
@@ -30,7 +30,7 @@ import (
 	"github.com/google/trillian/crypto/keys"
 	"github.com/google/trillian/extension"
 	"github.com/google/trillian/monitoring"
-	"github.com/google/trillian/quota"
+	mysqlq "github.com/google/trillian/quota/mysql"
```

Comment:
I'd rather stay clear of storage/mysql - there's too much code in that package already. That, along with Go's default package visibility, means things are getting a bit intertwined.

We could move to storage/mysql/quota, but I'm not sure exactly how you want it to be done (or why). What do you have a mind, a file with query constants? Also, if we're not sure about this, could we leave it to a separate PR? If we move the queries out it might make sense to rename some things (`sql.QuotaManager` instead of `mysql.QuotaManager`, for example).